test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -15,6 +15,190 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+
+    describe('createFunction', function(){
```

Comment:
spaces in function expression, `function () {` , This applies to all other functions too